cocoa/events_darwin.go

```diff
@@ -46,6 +46,11 @@ func (w *Window) EventChan() (events <-chan interface{}) {
 	downKeys := make(map[string]bool)
 	ec := make(chan interface{})
 	go func(ec chan<- interface{}) {
+
+		var noX int = 1<<31 - 1
```

Comment:
int32 if you're going to copy this weirdness with the bit offset. i honestly don't remember how this was arrived at.